### PR TITLE
NathanTech.Calliope version 3.6

### PR DESCRIPTION
**Pull request by Mudb0y**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56771)

### CODE REVIEW

File: manifests/n/NathanTech/Calliope/3.6/NathanTech.Calliope.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: NathanTech.Calliope
+PackageVersion: 3.6
+Installers:
+- Architecture: neutral
```
Comment (Trenly) [inline manifests/n/NathanTech/Calliope/3.6/NathanTech.Calliope.installer.yaml:7]: Installs to the program files x86 directory
```suggestion
- Architecture: x86
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Package does not install unattended
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/56771#pullrequestreview-930867395

**Pull request closed without merging**